### PR TITLE
Updated documentation based on FINOS requirements

### PR DESCRIPTION
**Pull request by ColinEberhardt**
See #24 

 - Added NOTICE - @maoo I've committed the email, I'm not keen on distributing email addresses where it is not entirely necessary
 - Added CONTRIBUTING.md  - @maoo I've significantly simplified this, I'm not keen on making contributors jump through hoops. My general preference is make life easier for first-timers!
 - Updated README - @maoo I've not added a roadmap, the issues are a more dynamic way of capturing this. Also, I've not added contribution instructions, instead pointing people to GitHub Guides if they need instructions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -83,3 +83,22 @@ If you'd like a demo of the current collection of hooks, you can do so by:
...
+ - If you want to contribute code, please raise a pull request. If you unsure of the process, please look at the [GitHub Guides](https://guides.github.com/).
+
+_NOTE:_ Commits and pull requests to FINOS repositories will only be accepted from those contributors with an active, executed Individual Contributor License Agreement (ICLA) with FINOS OR who are covered under an existing and active Corporate Contribution License Agreement (CCLA) executed with FINOS. Commits from individuals not covered under an ICLA or CCLA will be flagged and blocked by the FINOS Clabot tool. Please note that some CCLAs require individuals/employees to be explicitly named on the CCLA.
+
+*Need an ICLA? Unsure if you are covered under an existing CCLA? Email [help@finos.org](mailto:help@finos.org)*
+
+
+## License
+
+Copyright {yyyy} {name of copyright owner}
```
Comment (oriondean) [inline README.md]: Should this have a year and name?

Comment (maoo) [reply README.md]: Yes, same as the `NOTICE` file.

Comment (ColinEberhardt) [reply README.md]: :face palm: I was sure I'd updated that!

**Comment by ColinEberhardt** [comment]
Updated to supply date / name for template

**Pull request merged**